### ISSUE TITLE
[image_picker] in iOS14.6  camera return a wrong rotation image

### ISSUE THREAD

**Issue by zhao935210** [flutter/flutter#83783]
plugin name : image_picker 
plugin version: 0.8.0

device: iphoneX 
os: iOS 14.6

pick image by camera  return a  wrong rotation image

**Comment by TahaTesser** [comment]
Hi @zhao935210 
Are you using resizing the image? I can only reproduce the issue if I resize the image using the [official sample](https://pub.dev/packages/image_picker/example). 

Following steps below i can see image has wrong rotation

| <img src="https://user-images.githubusercontent.com/48603081/120485763-16948700-c3bd-11eb-8ad6-9acd7fe15435.PNG" height="450" /> | <img src="https://user-images.githubusercontent.com/48603081/120485772-185e4a80-c3bd-11eb-82a3-7156c2b60023.PNG" height="450" /> |<img src="https://user-images.githubusercontent.com/48603081/120485848-290ec080-c3bd-11eb-98ee-5d4d02414a5c.PNG" height="450" /> |




<details> 
<summary>flutter doctor -v</summary> 
	
```console
[✓] Flutter (Channel stable, 2.2.1, on macOS 11.4 20F71 darwin-x64, locale
    en-GB)
    • Flutter version 2.2.1 at /Users/tahatesser/Code/flutter_stable
    • Framework revision 02c026b03c (6 days ago), 2021-05-27 12:24:44 -0700
    • Engine revision 0fdb562ac8
    • Dart version 2.13.1

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Volumes/Extreme/SDK
    • Platform android-30, build-tools 30.0.3
    • ANDROID_HOME = /Volumes/Extreme/SDK
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer
    • Xcode 12.5, Build version 12E262
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)

[✓] VS Code (version 1.56.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.23.0

[✓] Connected device (3 available)
    • Taha’s iPhone (mobile) • 00008020-001059882212002E • ios            • iOS
      14.6
    • macOS (desktop)        • macos                     • darwin-x64     •
      macOS 11.4 20F71 darwin-x64
    • Chrome (web)           • chrome                    • web-javascript •
      Google Chrome 91.0.4472.77

• No issues found!
```

</details>

Related to https://github.com/flutter/flutter/issues/74014

**Comment by zhao935210** [comment]
yes, I resized the image.
Here is my code:


PickedFile? pickedFile = await ImagePicker().getImage(
  source: ImageSource.camera,
  maxWidth: 720,
  maxHeight: 1280,
  imageQuality: 90,
);




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Taha ***@***.***&gt;; 
发送时间: 2021年6月2日(星期三) 晚上9:35
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [flutter/flutter] [image_picker] in iOS14.6  camera return a wrong rotation image (#83783)





 
Hi @zhao935210
 Are you using resizing the image? I can only reproduce the issue if I resize the image using the official sample.
 
Following steps below i can see image has wrong rotation
 
|  |  | |
   flutter doctor -v  [✓] Flutter (Channel stable, 2.2.1, on macOS 11.4 20F71 darwin-x64, locale     en-GB)     • Flutter version 2.2.1 at /Users/tahatesser/Code/flutter_stable     • Framework revision 02c026b03c (6 days ago), 2021-05-27 12:24:44 -0700     • Engine revision 0fdb562ac8     • Dart version 2.13.1 [✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)     • Android SDK at /Volumes/Extreme/SDK     • Platform android-30, build-tools 30.0.3     • ANDROID_HOME = /Volumes/Extreme/SDK     • Java binary at: /Applications/Android       Studio.app/Contents/jre/jdk/Contents/Home/bin/java     • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)     • All Android licenses accepted. [✓] Xcode - develop for iOS and macOS     • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer     • Xcode 12.5, Build version 12E262     • CocoaPods version 1.10.1 [✓] Chrome - develop for the web     • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome [✓] Android Studio (version 4.2)     • Android Studio at /Applications/Android Studio.app/Contents     • Flutter plugin can be installed from:       🔨 https://plugins.jetbrains.com/plugin/9212-flutter     • Dart plugin can be installed from:       🔨 https://plugins.jetbrains.com/plugin/6351-dart     • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264) [✓] VS Code (version 1.56.2)     • VS Code at /Applications/Visual Studio Code.app/Contents     • Flutter extension version 3.23.0 [✓] Connected device (3 available)     • Taha’s iPhone (mobile) • 00008020-001059882212002E • ios            • iOS       14.6     • macOS (desktop)        • macos                     • darwin-x64     •       macOS 11.4 20F71 darwin-x64     • Chrome (web)           • chrome                    • web-javascript •       Google Chrome 91.0.4472.77 • No issues found!
  
Related to #74014

**Comment by TahaTesser** [comment]
I just reopened https://github.com/flutter/flutter/issues/74014
Closing as duplicate
